src/main/java/org/spburegistry/backend/service/UserService.java

```diff
@@ -1,9 +1,10 @@
 package org.spburegistry.backend.service;
 
-import org.spburegistry.backend.entity.User;
+import java.util.Set;
+
+import org.spburegistry.backend.dto.UserTO;
 
 public interface UserService {
-    public Iterable<User> findAll();
-    public User findById(long id);
-    public User findByName(String name);
+    public Set<UserTO> findAll();
```

Comment:
I think it is better to leave Iterable for interface declaration because it it more flexible than Set

Reply:
Done